.github/workflows/checks.yaml

```diff
@@ -7,6 +7,8 @@ on:
       - main
     paths:
       - '**/*.sh'
+      - .github/workflows/checks.yml
```

Comment:
wooo `checks.yaml` vs `checks.yml`, is that correct? 

If you add a self reference in the path, would not that create an infinite loop?

Reply:
> wooo checks.yaml vs checks.yml, is that correct?

That day when they allowed both suffixes to be used, someone was obviously sleeping 🤦🏻‍♂️
Thanks for catching this before the run would fail 👍 

> So IIUC, any change in this file will trigger it, correct?

Yes, i believe this is what we want, right?

Reply:
> Yes, i believe this is what we want, right?

Yes, yes, I just like to confirm what I understand :)

> That day when they allowed both suffixes to be used, someone was obviously sleeping 

yep...

But I think you still missed the change for that file, it's not marked as `outdated` by GH ;)